### PR TITLE
[plugin.video.watchmojo] 3.0.2

### PR DESCRIPTION
**Pull request by willforde**
### Description
Fixed some scraping issues that added some useless entries.
Move main code into a module, makes for a small entry point.

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.watchmojo/addon.py [hunk 1]
```diff
@@ -17,156 +17,12 @@
...
-    :param Resolver plugin: Tools related to Resolver callbacks.
-    :param unicode url: The url to a video.
-    :return: A playable video url.
-    """
-    url = url_constructor(url)
-    return plugin.extract_source(url)
+import sys
+import os
 
+lib = os.path.join(os.path.dirname(__file__), "resources", "lib")
```
Comment (enen92) [inline plugin.video.watchmojo/addon.py]: please properly turn this into a python module by creating `__init__` files along the paths you need to import. `sys.path.append` is not a clean way to import your dependencies.

File: plugin.video.watchmojo/resources/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+pass
```
Comment (enen92) [inline plugin.video.watchmojo/resources/__init__.py]: you don't need `pass`, empty files will do it

File: plugin.video.watchmojo/resources/lib/__init__.py [hunk 3]
```diff
@@ -0,0 +1 @@
+pass
```
Comment (enen92) [inline plugin.video.watchmojo/resources/lib/__init__.py]: same

**Comment by willforde** [comment]
O, good to know. I might as well make the change. Thanks.

**Pull request merged**